ror-tools/src/test/scala/tech/beshu/ror/tools/RorToolsAppTest.scala

```diff
@@ -0,0 +1,172 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.tools
+
+import org.scalatest.BeforeAndAfter
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.must.Matchers.include
+import org.scalatest.matchers.should.Matchers.{equal, should}
+import tech.beshu.ror.tools.RorToolsApp.Result
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, IOException, PrintStream}
+import java.nio.file.Files.*
+import java.nio.file.attribute.BasicFileAttributes
+import java.nio.file.{Files, Path, Paths}
+import java.util.UUID
+
+class RorToolsAppTest extends AnyFunSuite with BeforeAndAfter {
+
+  test("Patching is successful for ES installation that was not patched (with consent given in arg)") {
+    withTestDirectory("test-es-not-patched") { testDirectory =>
+      val (result, output) = captureResultAndOutput {
+        RorToolsApp.handle(Array("patch", "--I-understand-implications-of-ES-patching", "yes", "--es-path", testDirectory))
+      }
+      result should equal(Result.Success)
+      output should include(
```

Comment:
it'd be good to add some additional verification step. Eg. we could calculate the hash of some ES dirs before and after patching and checking if the hash is ok. 

It'd be good to have "There and Back Again" test too.